### PR TITLE
Add documentation for Phpactor

### PR DESCRIPTION
**Pull request by theLine**
This PR will add documentation for Phpactor language server

### CODE REVIEW

File: docs/src/language_servers.md [hunk 1]
```diff
@@ -311,8 +311,29 @@ Follow installation instructions on [LSP-promql](https://github.com/prometheus-c
...
 
+### Phpactor
+
+1. Install [Phpactor globally](https://phpactor.readthedocs.io/en/master/usage/standalone.html#installation-global).
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"phpactor"` client configuration to the `"clients"`:
+
+    ```json
+    {
+        "clients": {
+            "reason": {
```
Comment (rwols) [inline docs/src/language_servers.md:328]: This says `”reason”`, typo?

Comment (rchl) [inline docs/src/language_servers.md:328]: Update name?

Comment (theLine) [reply docs/src/language_servers.md]: Oops, I forgot to rename that, see commit b0c1045862601ee6e6fd3250d3a441d46a6e8d7c

**Review by rwols — APPROVED** [review]

File: docs/src/language_servers.md (hunk 1 above)
Comment (theLine) [reply docs/src/language_servers.md]: I'll resolve this conversation, because it's a duplicate one :)

**Pull request merged**